appcircle_audit_permission_changes/1.0.2/component.yaml

```diff
@@ -0,0 +1,31 @@
+platform: iOS
+buildPlatform:
+displayName: "Audit Permission Changes"
+description: "Check your permissions of app according to reference branch"
+docsUrl: https://github.com/appcircleio/appcircle-ios-permission-check-component
+requiredComponents: "appcircle_git_clone"
+precedingComponents: 
+followingComponents: "appcircle_export_build_artifacts"
```

Comment:
> @boztopuz Since this review is very repetitive, I will not give the same feedback in the rest, but can you provide the checks for all of them?

I mentioned the above situation in the previous review, but since the `export_build_artifacts` for which I did not add a comment were not corrected, I am adding them again for each one.
=> There is no any dependencies with `appcircle_export_build_artifacts`

Reply:
This component will write the permissions to text file and send it to output_dir